### PR TITLE
lnrpc+rpcserver: Adding timestamps to RoutingPolicy

### LINKED ISSUE #2310: Channel edges should have their own timestamps on each edge
### Background

Each node in a channel independently sets their policy on their own individual edge. But in the GRPC ChannelEdge and RoutingPolicy messages, only a single timestamp is returned.

Instead, each policy should have its own timestamp. These timestamps should be returned in the graph subscription to channels, in describe graph channels, and in individual channel lookups.

### Steps to reproduce

1. Lookup a channel
2. Look at the policy update time

### Expected behaviour

The policy update time of a channel edge should reflect the time the author of the policy update created the update.

### Actual behaviour

Sometimes no timestamp is returned, other times it is left ambiguous as to which policy has been updated at the related timestamp.

### PR DESCRIPTION
**Pull request by ccdle12**
fixes #2310

This is an early PR. 

This PR adds a `LastUpdate` field to the `RoutingPolicy` message in the `rpc.proto` file and subsequent generated files.

It also adds the `LastUpdate` field in the `rpcserver` and displays a unix timestamp when each party updates their channel policy.

Would appreciate any feedback with the following:
* I'm currently looking for tests to update according to these changes
* Do we also need to remove the LastUpdate field in the `ChannelEdge`?

### CODE REVIEW

File: lnrpc/rpc.proto [hunk 1]
```diff
@@ -1527,6 +1527,7 @@ message RoutingPolicy {
     int64 fee_base_msat = 3 [json_name = "fee_base_msat"];
     int64 fee_rate_milli_msat = 4 [json_name = "fee_rate_milli_msat"];
     bool disabled = 5 [json_name = "disabled"];
+    uint32 last_update = 6 [json_name = "last_update"];
```
Comment (halseth) [inline lnrpc/rpc.proto]: protos should be regenerated in this commit.

**Comment by halseth** [comment]
@ccdle12 What do you mean by "remove the LastUpdate field in the ChannelEdge?"

**Comment by cfromknecht** [comment]
@ccdle12 by chance are you using a different version of the proto compiler? The output formats look quite different

**Comment by ccdle12** [comment]
@cfromknecht My protoc version is currently :

`libprotoc 3.6.1`

Also strangely enough, when I run `gen_protos.sh` it creates a folder `github` in the `lnrpc` directory with all the newly generated protobuf files.

**Comment by ccdle12** [comment]
> @cfromknecht My protoc version is currently :
> 
> `libprotoc 3.6.1`
> 
> Also strangely enough, when I run `gen_protos.sh` it creates a folder `github` in the `lnrpc` directory with all the newly generated protobuf files.

@cfromknecht I was just going through the contribution guidelines and its states the `protoc` version being used is `v3.4.0`, if thats indeed the case as the most up to date recommended version, I can downgrade and then regenerate the files.

**Comment by wpaulino** [comment]
@ccdle12 for now `v3.4.0` is the one to use.

**Comment by ccdle12** [comment]
> @ccdle12 for now `v3.4.0` is the one to use.

@wpaulino thanks, the last fixup! adding the deprecated annotation was generated using `v3.4.0`

**Comment by wpaulino** [comment]
@ccdle12 I believe the other lnrpc changes weren't generated with `v3.4.0` as well.

**Comment by halseth** [comment]
Still looks like wrong proto compiler has been used.

**Comment by ccdle12** [comment]
@halseth thanks, when I run protoc --version:

```
$ protoc --version
$ libprotoc 3.4.0
```

I'll completely remove the binaries and follow the installation instructions again.

**Comment by halseth** [comment]
Also check that go-gen-proto and the other proto tools are the correct version

**Comment by ccdle12** [comment]
> Also check that go-gen-proto and the other proto tools are the correct version

Thanks @halseth, I completely removed all project dependencies and reinstalled. It seems like the proto files are being generated per `protoc v3.4.0`. There were updates to `go.mod` and `go.sum` although I'm not sure if those are normal considering the updates.

File: lnrpc/rpc.proto (hunk 1 above)
Comment (ccdle12) [reply lnrpc/rpc.proto]: fixed!

**Comment by ccdle12** [comment]
> Needs a squash and rebase, and regenerate the protos.
> 
> The `go.*` files should not be modified if done correctly 😀

@halseth 

Thanks for the review and so sorry it's been a while!

I've squashed and rebased according to the updated version of the proto file. 

I've also bumped the number of the `last_update` field in `RoutingPolicy` to 7, in light of the new additional field `max_htlc_msat` in `RoutingPolicy`.

**Comment by halseth** [comment]
@ccdle12 Still looks like your proto tools are not the correct version?

**Comment by ccdle12** [comment]
@halseth thanks for catching that, apologies for the oversight, I think I switched version to 2 for a different project. Should be regenerated with version 3.

**Comment by ccdle12** [comment]
@wpaulino @cfromknecht thanks for the review everyone, I'll keep an eye on the PR and keep rebasing until ready to merge

**Comment by cfromknecht** [comment]
@ccdle12 needs rebase since #3279 was merged

**Comment by ccdle12** [comment]
> @ccdle12 needs rebase since #3279 was merged

@cfromknecht thanks! rebased

**Comment by cfromknecht** [comment]
@ccdle12 :( needs another

**Pull request merged**